docs/docs/software-architecture.md

```diff
@@ -59,7 +58,7 @@ So, in the context of our application:
 
 A significant feature of this architecture is the flow of dependencies, which can be seen by the arrows moving in from the blue layer to the yellow layer in figure 1. This signifies that an outer layer can depend on an inner layer, but an inner layer cannot depend on an outer layer. The things that are most likely to change are kept on the outer layers and the things are less likely to change are kept on the inner most layers, helping the application stand the test of time. This makes the inner layers much more stable than the outer layers thus, the tools used to build the application can be modified easily (blue layer) but the core concepts and ideas behind the application are less likely to change (yellow layer). [3]
 
-
+   
 ## 3. Logical and Code Component Overview
 
 The logical view of Swapchain is comprised of 4 main components: User Interface, ACCS, Class, HTLC, and Verification.
```

Comment:
a crucial part here is (without comma in line 73 - kleinkariert muss man ja auch manchmal sein :-P